src/index.js

```diff
@@ -35,6 +35,7 @@ function ReactWordCloud({
 	options,
 	size: initialSize,
 	words,
+	...passThroughProps
```

Comment:
nit: rename to `rest` and same in line 67.

You might need to pull/rebase because I made changes a few minutes ago.

Reply:
Ah, yep sure, will update and rebase.  That's pushing now.